src/App.vue

```diff
@@ -100,7 +100,8 @@
       v-if="bannerText"
       tile
       dense
-      type="warning"
+      class="mb-0"
+      color="#fb8c00"
```

Comment:
Is it possible to move this into the styles?

Reply:
Yup. See latest commit.

Reply:
Awesome, thanks!